Pipfile

```diff
@@ -10,6 +10,7 @@ pillow = "*"
 imagehash = "*"
 mss = "*"
 gooey = "*"
+pygi = "*"
 
```

Comment:
```suggestion
pycairo = "*"
pygobject = "*"

```

Reply:
I added the dependency in 0288995f437fc35a5088bf3c91e5fcd628a0ccc4.